.github/workflows/ci-systemd-release.yml

```diff
@@ -16,8 +16,15 @@ permissions: read-all
 
 jobs:
   goreleaser:
-    runs-on: ubuntu-22.04
-    if: github.repository == 'kubearmor/kubearmor'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - arch: amd64
+            os: ubuntu-22.04
+          - arch: arm64
+            os: ubuntu-22.04-arm
+    # if: github.repository == 'kubearmor/kubearmor'
```

Comment:
This appears to be unintentionally commented out.

Reply:
yup.